README.rst

```diff
@@ -0,0 +1,183 @@
+.. contents:: Topics
+
+Overview
+========
+
+**Warning:** Subject to frequent updates
+       This is a "living document", expect it to change as we progress with the Collections work over the next few months.
+
+Feedback
+========
+
+As with any project it's very important that we get feedback from users, contributors and maintainers. We recognise that the move to Collections is a big change in how Ansible is developed and delivered. Early cuts, which maybe will (read will) be broken from time to time.
```

Comment:
> Early cuts, which maybe will (read will) be broken from time to time.

This line doesn't make sense to me... do you mean "Early attempts in devel will be broken from time to time" or something like that, or that this documentation will be broken from time to time?